implementation/.github/workflows/update-dependencies-from-metadata.yml

```diff
@@ -340,3 +340,22 @@ jobs:
           token: ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
           title: "Updates buildpack.toml with ${{ steps.update.outputs.new-versions }}"
           branch: automation/buildpack.toml/update-from-metadata
+
+  failure:
+    name: Alert on Failure
+    runs-on: ubuntu-22.04
+    needs: [ retrieve, get-compile-and-test, test, compile, update-metadata, assemble ]
+    if: ${{ always() && needs.retrieve.result == 'failure' || needs.get-compile-and-test.result == 'failure' || needs.test.result == 'failure' || needs.compile.result == 'failure' || needs.update-metadata.result == 'failure' || needs.assemble.result == 'failure' }}
+    steps:
+    - name: File Failure Alert Issue
+      uses: paketo-buildpacks/github-config/actions/issue/file@main
+      with:
+        token: ${{ secrets.PAKETO_BOT_GITHUB_TOKEN }}
```

Comment:
```suggestion
        token: ${{ secrets.GITHUB_TOKEN }}
```

Reply:
Also, I don't think the org-scoped token is needed for this action. The repo-scoped token should be ok.